pandas/_libs/interval.pyx

```diff
@@ -51,17 +51,38 @@ cdef class Interval(IntervalMixin):
 
     .. versionadded:: 0.20.0
 
-    Attributes
+    Parameters
     ----------
-    left, right : values
-        Left and right bounds for each interval.
+    left : value
+        Left bounds for interval.
+    right : value
+        Right bound for interval.
     closed : {'left', 'right', 'both', 'neither'}
         Whether the interval is closed on the left-side, right-side, both or
         neither. Defaults to 'right'.
 
+    Examples
+    --------
+    >>> abc = pd.Interval(left='a', right='c', closed='both')
+    >>> 'b' in abc
```

Comment:
I would also show the abc (so you see the repr)

(and personally I would find a integer (numeric) example better? (more common?))

Reply:
Ok, I see that, I've changed it